src/GUI/UI/Spellbook.cpp

```diff
@@ -266,12 +266,10 @@ void GUIWindow_Spellbook::Release() {
 
 void LoadSpellbook(unsigned int spell_school) {
```

Comment:
Can add TODO about using SPELL_SCHOOL type.

Reply:
It's not SPELL_SCHOOL though, because SPELL_SCHOOL that we have is actually DAMAGE_TYPE. Added a TODO.